src/libraries/System.DirectoryServices/src/System/DirectoryServices/SearchResultCollection.cs

```diff
@@ -214,6 +214,49 @@ void ICollection.CopyTo(Array array, int index)
             InnerList.CopyTo(array, index);
         }
 
+        /// <devdoc>
+        /// Supports a simple type-specific wrapper for the underlying cached list
+        /// </devdoc>
+        private sealed class AlreadyReadResultsEnumerator : IEnumerator
```

Comment:
>class [](http://example.com/codeflow?start=23&length=5)

can this be a struct?

Reply:
Good point, I will adjust after the Blues game.

Reply:
Switched to `struct`